credoai/lens.py

```diff
@@ -44,15 +44,87 @@ class CredoGovernance:
     """ Class to store governance data.
 
     This information is used to interact with the CredoAI
-    Governance Platform. Artifacts (AI solutions, models, and datasets)
-    are identified by a unique ID which can be found on the platform.
+    Governance Platform. Artifacts (AI solutions, model projects,
+    models, and datasets) are identified by a unique ID which 
+    can be found on the platform.
 
     To make use of the governance platform a .credo_config file must
     also be set up (see README)
     """
-    ai_solution_id: str = None
-    model_id: str = None
-    data_id: str = None
+    def __init__(self, 
+                ai_solution_id: str = None, 
+                project_id: str = None, 
+                model_id: str = None, 
+                data_id: str = None):
+
+        self.ai_solution_id = ai_solution_id
+        self.project_id = project_id
+        self.model_id = model_id
+        self.data_id = data_id
+        self.assessment_spec = {}
+
+    def get_assessment_spec(self):
+        """Get assessment spec
+        
+        If not retrieved yet, attempt to retrieve them from AI Governance platform
+        """
+        if not self.assessment_spec:
+            self.retrieve_assessment_spec()
+        spec = {}
+        metrics = self.assessment_spec
+        if metrics:           
+            spec['metrics'] = list(metrics.keys())
+        return {"FairnessBase": spec}
+
+    def get_info(self):
```

Comment:
Would be helpful to add docstring to this function

Reply:
Done